docs/sdk/v3/guides/liquidity/03-checking-collecting-fees.md

```diff
@@ -1,6 +1,57 @@
+
+At the end of the guide, given the inputs above, we should be able to mint a liquidity position with the press of a button and view the position's id on the UI of the web application. We should also be able to collect the accrued fees (if any) with the press of a button and see the change reflected in the balance of our tokens.
+
+## Example
+
+### Collecting accrued fees from our position
+
+All of the fee collecting logic can be found in the [`collectFees`](https://github.com/Uniswap/examples/blob/be67e7df220b0a270c9d18bbaab529e017213adf/v3-sdk/collecting-fees/src/example/Example.tsx#L24) function. The function is very similar to `mintPosition`, except for some details that we will go over. Notice how the **Collect Fees** button is disabled until a position is minted.
+
+Note that we do not need to give approval to the `NonfungiblePositionManager` to transfer our tokens as we have already done that when minting our position. 
+
+To start, we construct an options object of type  [`CollectOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L105) that holds the data about the fees we want to collect:
+
+```js reference title="Constructing the collectOptions object" 
+https://github.com/Uniswap/examples/blob/be67e7df220b0a270c9d18bbaab529e017213adf/v3-sdk/collecting-fees/src/example/Example.tsx#L31-L48
+```
+
+Similar to the other functions exposed by the `NonfungiblePositionManager`, we pass the `tokenId` which is just the `positionId` that we passed in as an argument to the function. We also provide the `recipient` of the fees, which in this case is our wallet's address.
+
+The other 2 parameters, `expectedCurrencyOwed0` and `expectedCurrencyOwed1`, concern the type of currency and **max** amount of currency we expect to get collect through accrued fees, one for each of the two tokens of the pool. The type of the parameters is `CurrencyAmount`, a type that describes both the type and amount of currency. The values we provide are configuration parameters to our guide.
+
+We then pass the options object to the `NonfungiblePositionManager`'s `collectCallParameters`'s function:
+
+```js reference title="Getting the calldata and value for the transaction" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/be67e7df220b0a270c9d18bbaab529e017213adf/v3-sdk/collecting-fees/src/example/Example.tsx#L51-L52
+```
+
+Now that we have both the calldata and value we needed for the transaction, we can build and execute the transaction:
+
+```js reference title="Building and submitting the transaction" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/be67e7df220b0a270c9d18bbaab529e017213adf/v3-sdk/collecting-fees/src/example/Example.tsx#L55-L64
+```
+
+After pressing the button, if someone has traded against our position, we should be able to note how the balance of USDC and DAI drops as we collect fees.
```

Comment:
Talking in slack but we should clarify what the currency of the fees will be, and if the USDC/DAI balance drops we should clarify why this happens quickly

Reply:
This is wrong - I meant to say **increase**